bw_simapro_csv/blocks/process.py

```diff
@@ -23,8 +23,7 @@
     "Waste treatment",
 }
 HAS_SUBCATEGORY = {
-    "Economic issues"
-    "Emissions to air",
+    "Economic issues" "Emissions to air",
```

Comment:
:eyes:

Reply:
Yeah, that was embarrassing. Got found by `pylint`, which is definitely picky, but also quite helpful sometimes.